source/index.html

```diff
@@ -4,9 +4,169 @@
     <meta charset="utf-8">
     <title>HTML Academy: {{projectTitle}}</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
-  </body>
+  <body class="page-body">
+    <header class="main-header">
+      <a href="#" class="main-header__logo">
+        <img class="main-header__logo-img" src="" alt="Логотип Кэт энерджи." width: 33.32px; height: 37.61px; >
+        <img src="" alt="Логотип Кэт энерджи." class="main-header__logo-img" >
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню.</span>
+        </button>
+      </nav>
+      <div class="main-nav__wrapper">
```

Comment:
```suggestion
      <div class="main-header__wrapper">
```
этот элекмент лежит в блоке main-header а не в main-nav
и вообще зачем тебе этот div.wrapper? ul вполне себе подойдет для списка - зачем пакет в пакете?